pkgs/games/humblebundle/antichamber.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, stdenv_32bit, lib, pkgsi686Linux, fetchHumbleBundle, unzip}:
+
+stdenv_32bit.mkDerivation rec {
```

Comment:
Why are you using `stdenv_32bit` here? You're already using `callPackage_i686` which already passes `pkgsi686Linux` function arguments. Thus `stdenv_32bit` is equal to `stdenv` (if no `callPackage_i686` was used, it would use multilib).